src/main/java/hw_jdi/forms/MetalsAndColorsForm.java

```diff
@@ -0,0 +1,73 @@
+package hw_jdi.forms;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import hw_jdi.entities.MetalsAndColors;
+
+
+public class MetalsAndColorsForm extends Form {
```

Comment:
This form should be parametrised by Entity class

Reply:
I  parametrised this form by MetalsAndColors class